### PR TITLE
Simplify experiments using `scope` prop in flow-bm

### PR DESCRIPTION
**Pull request by amitdahan**
### 🔦 Summary
Experiments are ready for consumption in pages & exported-components.
This is purely an internal change.

Spoke with @Gongreg about accessing experiments in module-level code (`moduleInit`, or in methods), sounds like it isn't necessarily the desired best-practice, so we'll leave this out for now.

### CODE REVIEW

File: packages/yoshi-flow-bm-runtime/src/experiments.tsx [hunk 1]
```diff
@@ -4,10 +4,8 @@ import { ExperimentsProvider } from '@wix/wix-experiments-react';
 export { useExperiments } from '@wix/wix-experiments-react';
 
 export function createExperimentsProvider(experimentsScopes: Array<string>) {
-  const experimentsProp = { scope: experimentsScopes };
-
   const Provider: FC = ({ children }) => (
-    <ExperimentsProvider options={experimentsProp}>
+    <ExperimentsProvider scope={experimentsScopes}>
```
Comment (yanivefraim) [inline packages/yoshi-flow-bm-runtime/src/experiments.tsx:8]: are you sure this is a valid API?
I cannot find `scope` as prop in the [docs](https://github.com/wix-private/fed-infra/tree/master/experiments/wix-experiments-react#usage) or in the [source](https://github.com/wix-private/fed-infra/blob/master/experiments/wix-experiments-react/src/index.ts#L8)

Comment (Gongreg) [reply packages/yoshi-flow-bm-runtime/src/experiments.tsx:8]: Hey, they are actually working on a branch: https://github.com/wix-private/fed-infra/tree/wix-react-experiments-16/experiments/wix-experiments-react

Comment (amitdahan) [reply packages/yoshi-flow-bm-runtime/src/experiments.tsx:8]: What @Gongreg said :D

Comment (yanivefraim) [reply packages/yoshi-flow-bm-runtime/src/experiments.tsx:8]: Oh, so this one will not work, until we merge, right?

Comment (amitdahan) [reply packages/yoshi-flow-bm-runtime/src/experiments.tsx:8]: Oh no, what he meant was, `wix-experiments-react` use a secondary branch `wix-react-experiments-16` to release the latest major `^4`.
They use `master` for `^3`.

**Pull request merged**